_articles/finding-users.md

```diff
@@ -19,15 +19,15 @@ related:
 
 ## Spreading the word
 
-There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. If you are hoping others will find and use your open source project, however, it's time to tell everybody about your hard work!
+There's no rule that says you have to promote an open source project when you launch. There are many fulfilling reasons to work in open source that have nothing to do with popularity. Instead of hoping others will find and use your open source project, you have to _spread the word_ about your hard work!
 
 ## Figure out your message
 
-Before you start the actual work of promoting your project, you should be able to explain what it does, and why it matters.
+Before you start the actual work of promoting your project, you should be able to explain what your project does, and why your project matters.
 
-What makes your project different or interesting? Why did you create it? Answering these questions for yourself will make it easier to convince others.
+What makes your project different or interesting? Why did you create it? Answering these questions for yourself will help you communicate your project's significance.
 
-Remember that people get involved as users, and eventually contributors, because it solves a problem for them. As you think about your project's message and value, try to view it through the lens of what _they_ might want.
```

Comment:
"try to view **it** through the lens" should be "try to view **them** through the lens" because the sentence is asking readers to view _two_ things: the project's message and the project's value.